### ISSUE TITLE
Housing Prices Neural Network classification

### ISSUE THREAD

**Issue by ayushijainn111** [UppuluriKalyani/ML-Nexus#631]
**Is your feature request related to a problem? Please describe.**
A regression problem using boston housepricing dataset in neural networks

**Describe the solution you'd like**
Predicting the house price using regression

**Describe alternatives you've considered**
I have considered the hyperparameters that are most suitable for the model along with incorporating learning rate.

**Approach to be followed (optional)**
Creating, compiling, fitting the model, checking the model summary and evaluating the model along with loss analysis.

**Additional context**
Assign me please! @Neilblaze

**Comment by UppuluriKalyani** [comment]
@ayushijainn111 it's already done try to raise something different
